modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/category/FormNavigatorExtensionsConfigurationCategory.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.configuration.admin.web.internal.category;
+
+import com.liferay.configuration.admin.category.ConfigurationCategory;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Jorge Ferrer
+ */
```

Comment:
@diegonvs didn't know you changed names, shouldn't this be your name here?

Reply:
I just copied and paste any of the *ConfigurationCategory.java classes and renamed. I can rename :)

Do We have a policy for authoring classes?

Reply:
@diegonvs I didn't find anything in our guidelines, but if you're creating a new file using your name is probably the way to go ... now since this is just a "copy", I don't think it's that important.
We should ask advice from @nhpatt, @javierdearcos, @LuismiBarcos and @izaera our Java gurus

Reply:
Yep, we do not have a policy that I am aware of but, even if we copy the code, we change the author because the old author is missing all the context of why the code is there.

Reply:
Makes senses, thanks @nhpatt

Reply:
Great! Makes sense! Changing :)

Reply:
e2894b3 ! :D